### PR TITLE
[THEMES] Add Reactive theme and Water wallpaper

### PR DESCRIPTION
**Pull request by jeffbox12**
## Purpose

Adds Reactive Theme Visual Style and a wallpaper called Water

This has been made to be familiar to users yet with a unique style with a color scheme that defines the image of ReactOS. It´s designed to be the default style of ReactOS.

- As of now, it only includes one builtin theme.

- There are some little issues that will be addressed later and are not high priority.

## Preview
reactive1
reactive2

### CODE REVIEW

**Comment by GeoB99** [comment]
This is ultra sick, nice job!

File: sdk/cmake/CMakeMacros.cmake [hunk 1]
```diff
@@ -225,6 +225,8 @@ macro(dir_to_num dir var)
         set(${var} 65)
     elseif(${dir} STREQUAL reactos/system32/spool/prtprocs/x64)
         set(${var} 66)
+	elseif(${dir} STREQUAL reactos/Resources/Themes/Reactive)
+        set(${var} 67)
```
Comment (learn-more) [inline sdk/cmake/CMakeMacros.cmake:229]: Needs to be added to `boot/bootdata/packages/reactos.dff.in`

**Review by Zaryob — COMMENTED** [review]
Wanna see in list of builtin themes

**Review by katahiromz — APPROVED** [review]
Please fix conflict.

**Comment by jeffbox12** [comment]
Because there are going to be some noticable visual changes, I will convert this PR to draft for the moment.

**Pull request closed without merging**